lua/lspconfig/configs/svelte.lua

```diff
@@ -1,11 +1,32 @@
 local util = require 'lspconfig.util'
 
+local function migrate_to_svelte_5()
+  local params = {
+    command = 'migrate_to_svelte_5',
+    arguments = { vim.uri_from_bufnr(0) },
+  }
+
+  local clients = util.get_lsp_clients({
+    bufnr = vim.api.nvim_get_current_buf(),
+    name = 'svelte',
+  })
+  for _, client in ipairs(clients) do
+    client.request('workspace/executeCommand', params, nil, 0)
```

Comment:
we have vim.lsp.buf.execute_command

Reply:
> I don't think the alternative (`client:exec_cmd`) is available for 0.9.

Not going to add lspconfig features for Nvim older than 0.11